nemo/collections/asr/modules/conv_asr.py

```diff
@@ -241,10 +239,6 @@ def _prepare_for_export(self):
             if isinstance(m, MaskedConv1d):
                 m.use_mask = False
                 m_count += 1
-
-            if isinstance(m, SqueezeExcite):
```

Comment:
Same as above.

Reply:
Why, if the masks are taken care of from forward() ?